api/api.go

```diff
@@ -461,6 +523,159 @@ func CreateOrb(ctx context.Context, logger *logger.Logger, namespace string, nam
 	return orb, err
 }
 
+func incOrbVersion(orbVersion string, segment string) (string, error) {
+	v, err := semver.NewVersion(orbVersion)
+	if err != nil {
+		return "", err
+	}
+
+	var v2 semver.Version
+	switch segment {
+	case "major":
+		v2 = v.IncMajor()
+	case "minor":
+		v2 = v.IncMinor()
+	case "patch":
+		v2 = v.IncPatch()
+	}
+
+	return v2.String(), nil
+}
+
+// IncrementOrb accepts an orb and segment to increment the orb.
+func IncrementOrb(ctx context.Context, logger *logger.Logger, configPath string, namespace string, orb string, segment string) (string, error) {
```

Comment:
IncrementOrb seems to be inconsistent with the other naming in this file. This should probably be `OrbIncrementVersion`.

I would use a  struct like `OrbIncrementVersionOptions` any time a function has more than 3 arguments.

Reply:
The args -> options struct is a great pattern that I'd like to apply to a larger refactor when the time comes.